R/inputs-webprocess.R

```diff
@@ -57,7 +57,7 @@ setMethod(f = "inputs",signature = "XMLAbstractDocument",
           definition = function(.Object, ...){
             inputXpath <- "//wps:Execute/wps:DataInputs/wps:Input"
             
-            inputs <- getNodeSet(.Object, inputXpath)
+            inputs <- getNodeSet(.Object, inputXpath, namespaces = pkg.env$NAMESPACES)
```

Comment:
I'm getting errors in most places where I add this (calls that use the default `namespaces`).